src/feature/hs/hs_config.c

```diff
@@ -577,16 +581,20 @@ config_service(const config_line_t *line, const or_options_t *options,
     tor_assert_nonfatal_unreached();
     goto err;
   }
+
```

Comment:
Lets not remove this whitespace. The reason is that the `if(ret < 0)` being glued to the `switch` condition is to reflect that they are linked together and that adding anything in between is probably a bad idea.